practice/batch/services/services_actions.rb

```diff
@@ -0,0 +1,18 @@
+module SERVICES
+  STOPPED = 'STOPPED'.freeze
+  RUNNING = 'RUNNING'.freeze
+
+
+  def self.service_status(service_name, status)
+    status_output = `SC query #{service_name}`
+    status_output.include?(status)
+  end
+
+  def self.service_start(service_name)
```

Comment:
The only thing we want to know after execution of this command - succeed it or not (true of false).
Also, it would be nice to verify at the beginning that the service isn't running.